modules/client/shared/src/main/scala/lucuma/itc/client/InstrumentMode.scala

```diff
@@ -18,176 +19,169 @@ import lucuma.core.enums.GmosNorthGrating
+          u  <- c.downField("fpu").as[GmosFpu.South]
+          d  <- c.downField("ccdMode").as[Option[GmosCcdMode]]
+          r  <- c.downField("roi").as[Option[GmosRoi]]
+        yield GmosSouthSpectroscopy(cw, g, f, u, d, r)
   }
 
   given Encoder[InstrumentMode] with
     def apply(a: InstrumentMode): Json =
       a match {
-        case a @ GmosNorthSpectroscopy(_, _, _, _, _) => Json.obj("gmosNSpectroscopy" -> a.asJson)
-        case a @ GmosSouthSpectroscopy(_, _, _, _, _) => Json.obj("gmosSSpectroscopy" -> a.asJson)
-        case a @ GmosNorthImaging(_)                  => Json.obj("gmosNImaging" -> a.asJson)
-        case a @ GmosSouthImaging(_)                  => Json.obj("gmosSImaging" -> a.asJson)
+        case a @ GmosNorthSpectroscopy(_, _, _, _, _, _) =>
+          Json.obj("gmosNSpectroscopy" -> a.asJson)
+        case a @ GmosSouthSpectroscopy(_, _, _, _, _, _) =>
+          Json.obj("gmosSSpectroscopy" -> a.asJson)
+        case a @ GmosNorthImaging(_, _)                  =>
+          Json.obj("gmosNImaging" -> a.asJson)
+        case a @ GmosSouthImaging(_, _)                  =>
+          Json.obj("gmosSImaging" -> a.asJson)
       }
 
   given Decoder[InstrumentMode] with
     def apply(c: HCursor): Decoder.Result[InstrumentMode] =
-      for {
+      for
         ns <- c.downField("gmosNSpectroscopy").as[Option[GmosNorthSpectroscopy]]
         ss <- c.downField("gmosSSpectroscopy").as[Option[GmosSouthSpectroscopy]]
         ni <- c.downField("gmosNImaging").as[Option[GmosNorthImaging]]
         si <- c.downField("gmosSImaging").as[Option[GmosSouthImaging]]
-        m  <- (ns, ss, ni, si) match {
+        m  <- (ns, ss, ni, si) match
                 case (Some(n), None, None, None) => (n: InstrumentMode).asRight
                 case (None, Some(s), None, None) => (s: InstrumentMode).asRight
                 case (None, None, Some(s), None) => (s: InstrumentMode).asRight
                 case (None, None, None, Some(s)) => (s: InstrumentMode).asRight
                 case _                           =>
                   DecodingFailure("Expected exactly one of 'gmosN' or 'gmosS'.", c.history).asLeft
-              }
-      } yield m
+      yield m
 
   given Eq[InstrumentMode] with
     def eqv(x: InstrumentMode, y: InstrumentMode): Boolean =
-      (x, y) match {
+      (x, y) match
         case (x0: GmosNorthSpectroscopy, y0: GmosNorthSpectroscopy) => x0 === y0
         case (x0: GmosSouthSpectroscopy, y0: GmosSouthSpectroscopy) => x0 === y0
         case (x0: GmosNorthImaging, y0: GmosNorthImaging)           => x0 === y0
         case (x0: GmosSouthImaging, y0: GmosSouthImaging)           => x0 === y0
         case _                                                      => false
-      }
 
-  final case class GmosNorthImaging(
-    filter: GmosNorthFilter
-  ) extends InstrumentMode
+  case class GmosNorthImaging(
+    filter:  GmosNorthFilter,
+    ccdMode: Option[GmosCcdMode]
```

Comment:
Why did you add a param here??

Reply:
Because I noticed in the ITC servlet that Gmos Imaging can have CCD options, at least binning:

<img width="775" alt="image" src="https://github.com/user-attachments/assets/7b54baed-ab0a-45db-a3c9-013049d1d527">

Reply:
How will thiis affect the odb which calls imaging to calculate acq times?

Reply:
The parameter is optional. So, for acquisition, the ODB can just omit the parameter, which is equivalent to what was being done until now.  This just paves the way for when we invoke ITC for science imaging.

Unless binning parameters should be passed for computing acquisition? Do you know if this is the case? @swalker2m? @andrewwstephens?